sale_view_adjust_cft/__openerp__.py

```diff
@@ -14,7 +14,10 @@
         'crm',
     ],
     "data": [
-        'security/crm_security.xml'
+        'security/crm_security.xml',
+        'views/crm_lead_views.xml',
```

Comment:
Let's make the file name `<model_name>_view.xml` instead of `<model_name>_views.xml`.  I think this is typo in the CONTRIBUTING document.